styles.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+@media (max-width: 600px) {
+  html {
+    font-size: 65%;
+  }
+}
+
+#welcome-section {
+  height: 100vh;
+  display: flex;
+}
+
+#navbar {
+  position: fixed;
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+}
```

Comment:
You have done a good job so far. I suggest you should add some more styles to your navigation like color, background, and position to make it more user-friendly.